src/mobot_client/mobot.py

```diff
@@ -328,13 +328,13 @@ def chat_router(message, match):
 
 
             elif active_drop.drop_type == DropType.ITEM:
```

Comment:
I'm confused by the nested if statements here checking for something very similar

Reply:
I cleaned a lot of this up. Can you take a look and see if it's more readable?